en/messages.json

```
@@ -383,7 +383,20 @@
   "btnContinue": {
     "message": "Continue",
     "description" : "Button Text, shown on an Error screen. Clicking it will continue the current flow."
+  },
+  "headlineAutoConnectOptions": {
+    "message": "Auto-connect options",
+    "description": "Headline for the Auto Connect options"
+  },
+  "labelDescribeAutoStartOnPBM":{
+    "message": "Always turn on VPN when using Private Browsing Mode"
+  },
+  "bodyAutoStartOnPBM": {
+    "message": " To turn this on, go to the add-ons menu, select “Mozilla VPN Extension”, and select “Allow” for “Run in Private Windows”.",
```

```suggestion
    "message": "To turn this on, go to the add-ons menu, select “Mozilla VPN Extension”, and select “Allow” for “Run in Private Windows”.",
```

Unnecessary space at the beginning?